Composer/packages/client/src/pages/design/createDialog.tsx

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import React, { useState, useContext } from 'react';
+import formatMessage from 'format-message';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { Stack, StackItem } from 'office-ui-fabric-react/lib/Stack';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+import { DialogCreationCopy } from '../../constants';
+import { DialogWrapper } from '../../components/DialogWrapper/index';
+import { StorageFolder } from '../../store/types';
+import { StoreContext } from '../../store';
+
+import { name, description, styles as wizardStyles } from './styles';
+
+interface CreateDialogParams {
```

Comment:
Also naming problem. `xxxParams` is not a meaningful name to a data type. Seems this object represents a collection of form fields (dialog creation form), maybe name it `Dialog` / `DialogData` / `DialogForm`.